Jenkinsfile-build-nodes

```diff
@@ -31,9 +32,9 @@ pipeline {
         steps {
           script {
             //Docker files are located in different directories
-            if (params.JENKINS_NODE == ("jenkins" || "jenkins-prod")) {
+            if (isJenkinsParentNode) {
               dir("./docker") {
-                sh "docker build ${dockerFileName} --pull --no-cache -t ${imageTag} ."
+                sh "docker build -f ${dockerFileName} --pull --no-cache -t ${imageTag} ."
```

Comment:
Was this command just broken before? The commit message just says that the flag has been added.

Reply:
Yes it was broken. If you specifically give the name of the docker file than this flag is needed: https://docs.docker.com/engine/reference/commandline/build/